apps/spruce/src/pages/adminSettings/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import { useRef, useState } from "react";
+import { H2 } from "@leafygreen-ui/typography";
+import { StickyHeaderContainer } from "components/Settings/sharedStyles";
+import { AdminSettingsTabRoutes } from "constants/routes";
+import { AdminSettings } from "gql/generated/types";
+import useIntersectionObserver from "hooks/useIntersectionObserver";
+import { AdminSaveButton } from "./AdminSaveButton";
+import { getTabTitle } from "./getTabTitle";
+
+interface Props {
+  adminSettingsData: AdminSettings;
+  tab: AdminSettingsTabRoutes;
+}
+
+export const Header: React.FC<Props> = ({ adminSettingsData, tab }) => {
+  const { title } = getTabTitle(tab);
+  const saveable = tab === AdminSettingsTabRoutes.General;
+
+  const headerScrollRef = useRef<HTMLDivElement>(null);
+  const [showShadow, setShowShadow] = useState(false);
+  useIntersectionObserver(headerScrollRef, ([entry]) => {
+    setShowShadow(!entry.isIntersecting);
+  });
+
+  return (
+    <>
+      <div ref={headerScrollRef} />
```

Comment:
This isn't critical but I noticed when clicking on a sidebar element, the element is now covered by the header once the scroll is complete. It would be nice to have this fixed, maybe in a separate ticket?